.github/workflows/spec.yml

```diff
@@ -11,19 +11,19 @@ on:
       - '*-dev'
 
 jobs:
-  madoko-lint:
+  asciidoc -lint:
```

Comment:
I believe the space between  `asciidoc` and `-lint` is causing this failure: https://github.com/p4lang/p4runtime/actions/runs/12320523043